scripts/requirements.txt

```
@@ -0,0 +1,12 @@
+# -- Requirements for tuple_tree_generator
+jsonschema
+PyYAML
+Jinja2
+# Needed for python < 3.9
+graphlib-backport
+
+# -- Requirements for revng_merge_dynamic
+pyelftools
+
+# -- Requirements for monotone-framework
+pygraphviz
```

Due to SA2:
```suggestion
networkx
pydot
```

Can you please make a companion PR for orchestra adding jinja2 as a dependency?

---

https://github.com/revng/orchestra/pull/62/